trunk/SUAVE/.idea/SUAVE.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
```

Comment:
necessary?

Reply:
I'm not sure where these changes came from. I think its part of the new python we are pulling in